### ISSUE TITLE
Wrong icon downloaded

### ISSUE THREAD

**Issue by teddythinh** [elrumo/macOS_Big_Sur_icons_replacements#1464]
### Describe your issue here, and if possible, add an image.

As I search for the Microsoft Word icon and download it from `ken_guru`, it downloads the Audacity icon.
![image](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85171754/6f52ecc0-ee30-4b76-b2b1-f64a7b21cc1f)
![image](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85171754/4b5be229-531f-46fe-962c-8e399ad0c61d)

**Comment by bootlegseahorse** [comment]
Yep, I also noticed a few wrong icons, noticeably several of the Python Idle would link to the icons of Python Launcher.
